cpu/sam3/include/periph_cpu.h

```diff
@@ -60,16 +60,25 @@ typedef uint32_t gpio_t;
  */
 #define CPUID_LEN           (16U)
 
+
 /**
- * @brief Override values for pull register configuration
+ * @brief   Override GPIO modes
+ *
+ * We use 4 bit to determine the pin functions:
```

Comment:
Change these to use preprocessor macros or something symbolic as the current implementation contains magic numbers in the code.